scripts/create-sample-worksheet.py

```diff
@@ -0,0 +1,383 @@
+import argparse
+import os
+import random
+import string
+import sys
+import time
+
+sys.path.append('..')
+from stress_test import cleanup
```

Comment:
Can we put `cleanup` and `run_command` in a separate utilities file?  Creating a worksheet shouldn't depend on tests.